src/test/java/uk/nhs/prm/e2etests/test/RepositoryE2ETest.java

```diff
@@ -555,4 +553,20 @@ private void setManagingOrganisationToRepo(String nhsNumber) {
             pdsAdaptorService.updateManagingOrganisation(nhsNumber, repoOdsCode, pdsResponse.getRecordETag());
         }
     }
+
+    @Test
+    void shouldSendUnexpectedMessageFormatsThroughToEhrTransferServiceDeadLetterQueue() {
+        String text = "Hello World!";
+        String sql = "SELECT * FROM Fragment";
+        String html = "<html><body><h1>This is html!</body></html>";
+        String binaryStream = "100110 111010 001011 101001";
+        String emptyJson = "{}";
+        String randomUUID = UUID.randomUUID().toString();
+        String[] messages = {text, sql, html, binaryStream, randomUUID, emptyJson};
+
+        for (String message: messages) {
+            mhsInboundQueue.sendUnexpectedMessage(message);
+            assertThat(ehrTransferServiceParsingDeadLetterQueue.getMessageContaining(message)).isNotNull();
+        }
```

Comment:
Could we just have the parameters in a collection and iterate through them? 😄 Similar to:

```suggestion
        final List<String> unexpectedMessages = List.of(
                "Hello World!",
                "SELECT * FROM Fragment",
                "<html><body><h1>This is html!</body></html>",
                "100110 111010 001011 101001",
                "{}",
                UUID.randomUUID().toString()
        );

        unexpectedMessages.forEach(message -> {
            mhsInboundQueue.sendUnexpectedMessage(message);
            assertThat(ehrTransferServiceParsingDeadLetterQueue.getMessageContaining(message)).isNotNull();
        });
    }
```